app/models/job.rb

```diff
@@ -67,4 +67,8 @@ def duration_seconds
   def extraction_folder_size_in_bytes
     `du #{extraction_folder} | cut -f1`.to_i
   end
+
+  def name
```

Comment:
Why is the job name a date?  Do you want doc comments like the other methods?

Reply:
The date is the only identifying information we have at the moment about the job and we show the date it was run on the front end :)

Reply:
Is 'name' a misnomer though?  Just wondering if it is actually a name or not.  And if it is a formatted date, is that view logic, or model logic?  Up to you, it just seemed a bit odd to me :)

Reply:
That's fair :). I actually made this method so that I could get the name for the job in a rails form helper but I am not using that anymore so I will get rid of this method